runtime/stdlib/flow.go

```diff
@@ -347,6 +347,12 @@ var AccountKeyAddedEventType = newFlowEventType(
 	AccountEventPublicKeyParameter,
 )
 
+var AccountKeyGetEventType = newFlowEventType(
+	"AccountKeyRetrieved",
+	AccountEventAddressParameter,
+	AccountEventPublicKeyParameter,
+)
```

Comment:
I'm not sure it's a good idea to emit an even each time a key is read, maybe remove this